src/Hipace.cpp

```diff
@@ -1413,6 +1482,30 @@ Hipace::Notify (const int step, const int it,
         MPI_Isend(psend_buffer, np_total, one_particle_size,
                   (m_rank_z-1+m_numprocs_z)%m_numprocs_z, loc_pcomm_z_tag, m_comm_z, loc_psend_request);
     }
+
+    // Send laser data
+    {
+        if (only_ghost) return;
+        if (!m_laser.m_use_laser) return;
+        const amrex::FArrayBox& laser_fab = m_laser.getFAB();
+        // Note that there is only one local Box in slice multifab's boxarray.
+        // const int box_index = laser_fab.IndexArray();
```

Comment:
```suggestion
        // Note that there is only one local Box in slice multifab's boxarray.
        // const int box_index = laser_fab.IndexArray();
```

Are these comments outdated?

Reply:
probably